packages/caisar-ir/caisar-ir.0.2/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis: "CAISAR's intermediate representation"
+maintainer: [
+  "LAISER team, Software Safety and Security Laboratory, CEA-List"
+]
+authors: ["LAISER team, Software Safety and Security Laboratory, CEA-List"]
+license: "LGPL-2.1-only"
+homepage: "https://git.frama-c.com/pub/caisar"
+doc: "https://git.frama-c.com/pub/caisar"
+bug-reports: "https://git.frama-c.com/pub/caisar/issues"
+depends: [
+  "ocaml" {>= "4.13"}
+  "dune" {>= "2.9" & >= "2.9.3"}
```

Comment:
```suggestion
  "dune" {>= "2.9.3"}
```
(This is fixed by removing dune as a dependency in the dune file, dune adds itself automatically. In this case it would add 2.9 probably though)